### PR TITLE
[#699] Updated heading background images on .simplelayout

### LINKED ISSUE #699: Images supposed to be in background make the layout resize
Images that should be just background decoration make the content change position when the image finishes loading.

Example, contact page: https://developers.italia.it/en/contacts
Peek 16-09-2020 15-20

The content should not change position while the image loads.

### PR DESCRIPTION
**Pull request by Laischon**
## Description
As already mentioned in # 699, this PR corrects the display of the images present at the beginning of the "Simplelayout".

Such as?
Added a `div` that contains a background image, i.e. the header image. This `div` has replaced the `img` tag previously present and has a minimum height of 405px and is responsive on width.

**Before:** image_before_1

**After:** 
image_after_1

## Checklist

- [ x ] I followed the indications in the [CONTRIBUTING](../CONTRIBUTING.md)
- [ x ] The documentation related to the proposed change has been updated accordingly (also comments in code).
- [ x ] Have you written new tests for your core changes, as applicable?
- [ x ] Have you successfully ran tests with your changes locally?
- [ x ] Ready for review! :rocket:

## Fixes

- Fixes #699

### CODE REVIEW

File: assets/scss/main.scss [hunk 1]
```diff
@@ -1033,6 +1033,21 @@ body {
...
+    }
+    &-main-image-container {
+        max-width: 100%;
+        height: auto;
+        background-size: cover;
+        background-repeat: no-repeat;
+        background-position: center;
+        margin: 0;
+        @include bootstrap.media-breakpoint-up(lg) {
+          margin: 0 15%;
```
Comment (bfabio) [inline assets/scss/main.scss:1049]:
```suggestion
            margin: 0 15%;
```

**Review by bfabio — CHANGES_REQUESTED** [review]
This doesn't cover the whole viewport at my resolution:

image

whereas before it did:

image

is this on purpose?

**Review by Laischon — COMMENTED** [review]
Changed media rule for images on viewport with a width lower than 1600px

**Review by bfabio — APPROVED** [review]

**Review by sebbalex — APPROVED** [review]

**Comment by bfabio** [comment]
@Laischon Thanks, merged! :rocket:

**Pull request merged**